Makefile

```diff
@@ -32,6 +32,9 @@ test-unit: venv
 test-integration: venv
 	CACHI2_GENERATE_TEST_DATA=$(GENERATE_TEST_DATA) venv/bin/tox -e integration
 
+pypiserver-run:
+	tests/pypiserver/start.sh
+
```

Comment:
I'm still not convinced by this target, even more so if with all the latest changes this was reduced to a mere invocation of a script. I'd encourage it if the pypi server were to serve as a development deployment of a crucial component to the project, but it's not, we only need it for a specific test which I expect will be the only use case for a long time. If anything, we could document this as part of the integration test suite section mentioning that one can use the script to run a server locally.

Reply:
Ack, removed and added to the README instead